app/upgrades.go

```diff
@@ -54,25 +54,28 @@ func (app UmeeApp) RegisterUpgradeHandlers() {
 	app.registerUpgrade6_4(upgradeInfo)
 	app.registerUpgrade("v6.5", upgradeInfo, nil, nil, nil)
 
-	app.registerUpgrade6_6(upgradeInfo)
+	// this upgrade is only for testnet (canon-4) network
+	app.registerUpgrade6_6_rc1(upgradeInfo)
 }
 
-func (app *UmeeApp) registerUpgrade6_6(upgradeInfo upgradetypes.Plan) {
-	planName := "v6.6"
+func (app *UmeeApp) registerUpgrade6_6_rc1(upgradeInfo upgradetypes.Plan) {
+	planName := "v6.6-rc1"
```

Comment:
I added planName as `v6.6-rc1` instead of `v6.6` because we will change auction module param values and rewards params for mainnet , so if any validator uses this rc1 release it will not impact the mainnet